CHANGELOG.md

```diff
@@ -177,7 +177,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where the group tree was not updated correctly after an entry was changed. https://github.com/JabRef/jabref/issues/3618
 - We fixed an issue where a right-click in the main table selected a wrong entry. https://github.com/JabRef/jabref/issues/3267
 - We fixed an issue where in rare cases entries where overlayed in the main table. https://github.com/JabRef/jabref/issues/3281
-- We fixed an issue where selecting a group messed up the focus of the main table / entry editor. https://github.com/JabRef/jabref/issues/3367
+- We fixed an issue where selecting a group messed up the focus of the main table/entry editor. https://github.com/JabRef/jabref/issues/3367
```

Comment:
"main table" and "entry editor" are two different things in JabRef. Not sure whether the "table/entry" is read together by the ready with your fix. Maybe reword? -- "main table and the entry editor"?

Reply:
yes what you suggested would be a batter option